### DISCUSSION TITLE
Filtro per data non funzionante

### DISCUSSION THREAD

**Discussion by spokke** [fattureincloud/api#334] [Q&A] [answered]
salve è possibile che il filtro sql YEAR(date) = x non funzioni sugli issued documents passandoglielo come query ? 
se viene passato in end con altro parametro, che da solo funziona' restituisce 👍 
{
  "error": {
    "message": "Invalid query syntax.",
    "validation_result": null
  }
}
se viene passato da solo rimane va in timeout 

grazie

**Comment by emajo** [comment, marked as answer]
Ciao @spokke, le funzioni SQL come `YEAR` non funzionano con i nostri filtri, solo la sintassi è quella di SQL, per cui per filtrare solo i documenti relativi ad un determinato anno il filtro da utilizzare è il seguente: `date >= '2023-01-01' and date <= '2023-12-31'`.

Grazie,
Emanuele.

**Comment by spokke** [comment]
ok grazie mille buon lavoro
